.github/ISSUE_TEMPLATE/bug_python.md

```diff
@@ -10,25 +10,35 @@ Thank you for reporting a bug! Please make sure you have searched for similar is
 By opening an issue, you agree with Atoti's terms of use and privacy policy available at https://www.atoti.io/terms and https://www.atoti.io/privacy-policy
 -->
 
-## Description
+## Steps to reproduce
 
 <!--
-Describe the bug clearly and concisely.
-Include the error message if you have one or specify the expected behavior.
+Include a concise notebook and dataset with the issue.
+If possible, indicate which cell triggers the error.
```

Comment:
Do we have to say notebook ? I prefer a having the code to reproduce directly in the issue if possible

Reply:
I just used the same sentence than before. I can say "code sample" rather than "notebook"